src/scripts/active-inactive-orgunit/index.js

```diff
@@ -0,0 +1,351 @@
+var request = require('request');
+var dateFormatter = require('./dateFormatter.js');
+const basicAuth = require("basic-authorization-header");
+
+var fs = require('fs')
+fs.readFile('config.json', 'utf8', function (err, conf) {
+    if (err) {
+        return console.error("Invalid config.json ", err);
+    }
+    var organisationUnitActivator = new OrganisationUnitActivator(JSON.parse(conf));
+    organisationUnitActivator.run();
+});
+
+/**
+ * Main class
+ */
+function OrganisationUnitActivator(conf) {
+    this.endpoints = {
+        DATAELEMENTGROUP: "/dataElementGroups/UID?fields=dataElements[id,attributeValues]",
+        ORGANISATION_UNITS_BY_UID: "/organisationUnits/UID?fields=level",
+        ORGANISATION_UNITS_BY_ORGANISATION_UNIT_GROUP: "/organisationUnitGroups/UID?fields=organisationUnits[id,closedDate,openingDate]",
+        ORGANISATION_UNITS_BY_PARENT_AND_LEVEL: "/organisationUnits/UID?level=LEVELVALUE&fields=id,closedDate,openingDate",
+        DATAVALUE_SETS: "/dataValueSets",
+        DATAVALUES_RESULT: "/dataValueSets?orgUnit=ORGUNIT&dataElementGroup=DATAELEMENTGROUP&startDate=1900-01-01&endDate=3017-07-01"
```

Comment:
@idelcano This is only used for logging purposes I think. Does it make sense? Do we actually need it?